.github/workflows/sync_crowdin.yml

```diff
@@ -38,16 +40,15 @@ jobs:
       - name: Remove empty values from JSON
         run: |
           cd assets/i18n
-          sudo chmod 777 *
           dart nuke.dart >> $GITHUB_STEP_SUMMARY
 
-      - name: Push out changes to i10n
+      - name: Commit translations
         run: |
           git config user.name revanced-bot
```

Comment:
Instead of setting up the user like this, we have a token for @revanced-bot That can be used instead, if necessary

Reply:
It's not necessary to use the token, the token would only be used for push permissions which this workflow already has, we would still have to set the git username